linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -72,9 +72,10 @@ class HttpInitializer extends ProtocolInitializer.Simple {
   protected val defaultServer = {
     val stk = Http.server.stack
       .replace(HttpTraceInitializer.role, HttpTraceInitializer.serverModule)
-      .replace(Headers.Ctx.serverModule.role, Headers.Ctx.serverModule)
+      .remove(Headers.Ctx.serverModule.role)
       .prepend(http.ErrorResponder.module)
       .prepend(http.StatusCodeStatsFilter.module)
+      .prepend(Headers.Ctx.serverModule)
```

Comment:
Would you mind adding a comment here that the Ctx.serverModule needs to be before the ErrorResponder module so that errors responses from the ErrorResponder will be cleared when `clearContext` is set?

Reply:
Hi, @adleong! Yes, I'll add the comment.